css/src/base.less

```diff
@@ -453,6 +453,22 @@ table.floatThead-table {
     margin: 10px;
 }
 
+.task-status-ok {
+  color: #9da827;
+}
+
+.task-status-submitted {
+  color: #3498db;
+}
+
+.task-status-wrong {
+  color: #f39c12;
+}
```

Comment:
Ak tato farba nie je definovana niekde v nasich design dococh, tak do nej pridaj trocha cervenej / uber zelenu. Ak mas protanopiu alebo nejaku podobnu farboslepost tak je to nerozoznatelne od `task-status-ok` ak ich nemas hned pri sebe.

Reply:
Musí to byť naša farba, keďže som si ju nevymyslel 😂

Reply:
Mám zmeniť brand-warning na niečo viac odlišné, keďže to vyplýva z toho?

Reply:
Vieme mat rozne ikonky pre rozne stavy?

Reply:
Dalo by sa to, ale výber glyphicons je dosť slabý :/

Reply:
Uprimne tie ikonky a farby mi toho moc nehovoria ked sa len tak na to pozrem. Co keby sme to rozdelili na 2 stlpce - stav popisu a stav programu? A potom mali `-ok-circle` ak je vsetko vpohode, `-time` ak sa caka na opravenie/sa testuje, `-ban-circle` ak nie je odovzdany, `-remove-circle` ak je program nedostal OK? Kazdy stav by zaroven mohol mat aj farbu, ale ikonka aj bez farby by definovala stav, cize by ani nebol problem s farboslepostou.